src/Tribe/Main.php

```diff
@@ -584,6 +586,32 @@ protected function loadLibraries() {
 			}
 		}
 
+		/**
+		 * Prevents Image Widget Plus weird version of Tribe Common Lib to
+		 * conflict with The Events Calendar
+		 *
+		 * It will make IW+ not load on version 1.0.2
+		 *
+		 * @since   4.8.1
+		 *
+		 * @return  void
+		 */
+		private function compatibility_unload_iwplus_v102() {
+			if ( ! class_exists( 'Tribe__Image__Plus__Main' ) ) {
+				return;
+			}
+
+			if ( ! defined( 'Tribe__Image__Plus__Main::VERSION' ) ) {
+				return;
+			}
+
+			if ( ! version_compare( Tribe__Image__Plus__Main::VERSION, '1.0.2', '<=' ) ) {
+				return;
+			}
+
+			remove_action( 'plugins_loaded', array( Tribe__Image__Plus__Main::instance(), 'plugins_loaded' ), 0 );
```

Comment:
Do we have time to add a notice like we do when we shutdown because of an older version of ET: https://github.com/moderntribe/the-events-calendar/blob/c68d9c6280c499503072b2d5ffdbf0f383af786a/src/Tribe/Main.php#L1449

That way at least we have some warning to people that IW+ is not running.

Reply:
I wanted to avoid any notices. Since we are going to release IW+ hotfix today I just go a sign-off from Zach that we dont need it if we can get the release on the same day.